### PR TITLE
Orbital: Use billing_address name as fallback

### PR DESCRIPTION
**Pull request by curiousepic**
When adding the AVSName element, if there is no name on the payment
method itself, look in the billing address hash for a name instead.

Remote:
68 tests, 316 assertions, 0 failures, 0 errors, 0 pendings, 0 omissions, 0 notifications
100% passed

Unit:
120 tests, 701 assertions, 0 failures, 0 errors, 0 pendings, 0 omissions, 0 notifications
100% passed

### CODE REVIEW

**Comment by curiousepic** [comment]
@tatsianaclifton had to recreate a PR after refactoring overlapping code from a prior merge, but I impelmented the suggestions you made.

**Comment by curiousepic** [comment]
Actually, @jessiagee since you touched related code, want to make sure you sign off on this. I tried using the  `billing_name` method I added for the echeck AVSname, but that set off a bunch of tests. But if it makes sense to utilize it for echecks instead of the current method,  let me know and I'll see what I can do.

**Comment by tatsianaclifton** [comment]
It looks good to me, but I'll wait on @jessiagee response before approving

**Review by tatsianaclifton — APPROVED** [review]
I approve. It's up to you if you want to wait on @jessiagee opinion also, but LGTM

**Comment by jessiagee** [comment]
Oh sorry @curiousepic for some reason I missed this alert entirely, going to 👀  it now!

**Review by jessiagee — APPROVED** [review]
Looks good and tests pass!

**Pull request closed without merging**
